LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-expected.txt

```diff
@@ -1,49 +1,30 @@
+CONSOLE MESSAGE: Refused to connect to http://localhost:8800/common/blank.html because it does not appear in the connect-src directive of the Content Security Policy.
 CONSOLE MESSAGE: This requires a TrustedScriptURL value else it violates the following Content Security Policy directive: "require-trusted-types-for 'script'"
+CONSOLE MESSAGE: Refused to connect to http://localhost:8800/common/blank.html because it does not appear in the connect-src directive of the Content Security Policy.
 CONSOLE MESSAGE: This requires a TrustedScriptURL value else it violates the following Content Security Policy directive: "require-trusted-types-for 'script'"
+CONSOLE MESSAGE: Refused to connect to http://localhost:8800/common/blank.html because it does not appear in the connect-src directive of the Content Security Policy.
 CONSOLE MESSAGE: This requires a TrustedScript value else it violates the following Content Security Policy directive: "require-trusted-types-for 'script'"
+CONSOLE MESSAGE: Refused to connect to http://localhost:8800/common/blank.html because it does not appear in the connect-src directive of the Content Security Policy.
 CONSOLE MESSAGE: This requires a TrustedScript value else it violates the following Content Security Policy directive: "require-trusted-types-for 'script'"
-CONSOLE MESSAGE: Refused to load http://localhost:8800/trusted-types/dummy.webm because it does not appear in the object-src directive of the Content Security Policy.
-CONSOLE MESSAGE: This requires a TrustedScript value else it violates the following Content Security Policy directive: "require-trusted-types-for 'script'"
+CONSOLE MESSAGE: Refused to connect to http://localhost:8800/common/blank.html because it does not appear in the connect-src directive of the Content Security Policy.
 CONSOLE MESSAGE: This requires a TrustedScriptURL value else it violates the following Content Security Policy directive: "require-trusted-types-for 'script'"
+CONSOLE MESSAGE: Refused to connect to http://localhost:8800/common/blank.html because it does not appear in the connect-src directive of the Content Security Policy.
 CONSOLE MESSAGE: This requires a TrustedScriptURL value else it violates the following Content Security Policy directive: "require-trusted-types-for 'script'"
-CONSOLE MESSAGE: Refused to load http://localhost:8800/trusted-types/dummy.webm because it does not appear in the object-src directive of the Content Security Policy.
-
-abc
-
+CONSOLE MESSAGE: Refused to connect to http://localhost:8800/common/blank.html because it does not appear in the connect-src directive of the Content Security Policy.
 
 PASS Trusted Type violation report: creating a forbidden policy.
-PASS Trusted Type violation report: creating a report-only-forbidden policy.
+FAIL Trusted Type violation report: creating a report-only-forbidden policy. assert_true: expected true got false
```

Comment:
See https://bugs.webkit.org/show_bug.cgi?id=286917 for bug to track addressing this. There seems to be a flaw in the logic for whether CSP allows a specific policy name.

Reply:
This bug is addressed by #39934